lib/course.rb

```diff
@@ -0,0 +1,23 @@
+class Course
+
```

Comment:
There is entirely too much whitespace in this file, line breaks should only be one at a time, and there should not be extra whitespace inside methods or between methods